### PR TITLE
HWKAPM-489 Move the three top level tabs as sub tabs under a single A…

### PR DESCRIPTION
**Pull request by objectiser**
…pplication Performance tab

### CODE REVIEW

**Comment by objectiser** [comment]
@ammendonca  Hi Alex, would you be able to do a review on this change?

After the project rename and scope change, I thought it best for the three areas to be under a single main tab called Application Performance.

I've done the minimal changes to enable this to work in the UI (I believe) but this leaves the code in a slightly misleading structure - but thought this could be handled as a separate issue, as I would like to get a release out this week.

In terms of the restructure, I will create a separate jira, and would appreciate your advice on how best to reorganise the code.

File: ui/src/main/scripts/plugins/apm/ts/apmPlugin.ts [hunk 1]
```diff
@@ -30,14 +30,75 @@ module APM {
...
+      }).
+      when('/hawkular-ui/apm/tracing', {
+        templateUrl: 'plugins/e2e/html/e2e.html',
+        controller: 'E2E.E2EController'
+      }).
+      when('/hawkular-ui/apm/btm', {
+        templateUrl: 'plugins/btm/html/btm.html',
+        controller: 'BTM.BTMController'
+      }).
+      when('/hawkular-ui/apm/btm', {
```
Comment (ammendonca) [inline ui/src/main/scripts/plugins/apm/ts/apmPlugin.ts]: This block is duplicated from above

Comment (objectiser) [inline ui/src/main/scripts/plugins/apm/ts/apmPlugin.ts]: I think the problem was the identification of the controller. How can it be specified with the sub path?

I tried defining the controller in the home but that did not work.

Comment (ammendonca) [inline ui/src/main/scripts/plugins/apm/ts/apmPlugin.ts]: Not sure what is the exact issue you're mentioning, just noted that lines 49-52 are exactly the same as 53-56.

What subpath are you trying to specify the controller for ?

Comment (objectiser) [inline ui/src/main/scripts/plugins/apm/ts/apmPlugin.ts]: Ah sorry, misunderstood. Thought you were referring to the fact that the subpath (line 33-35) were being duplicated by the route/when code.

I'll remove the duplicate route.

**Comment by ammendonca** [comment]
LGTM

**Comment by objectiser** [comment]
Thanks Alexandre!

**Pull request merged**